sage.py

```diff
@@ -1629,126 +1425,111 @@ def make_AG(condensed_v_data, condensed_data, state_groups, sev_sinks, datafile,
     #    print(attackerID, len(v), set([x.split('|')[0] for x in v]))
 
 
-
-
-## ----- main ------    
+# ----- MAIN ------
 
 if len(sys.argv) < 5:
-    print('USAGE: sage.py {path/to/json/files} {experiment_name} {alert_filtering_window (def=1.0)} {alert_aggr_window (def=150)} {(start_hour,end_hour)[Optional]}')
+    print('Usage: sage.py {path/to/json/files} {experiment_name} {alert_filtering_window (def=1.0)} {alert_aggr_window (def=150)} {(start_hour,end_hour)[Optional]}')
     sys.exit()
 
-folder = sys.argv[1]
-expname = sys.argv[2]
-t = float(sys.argv[3])
-w = int(sys.argv[4])
-_s_,_e_ = 0, 100
+path_to_json_files = sys.argv[1]
```

Comment:
I have renamed these variables to the ones defined in "Usage", since they are easier to understand